sdk/go/common/util/ciutil/vars_test.go

```diff
@@ -142,3 +142,18 @@ func TestDetectVarsDisableCIDetection(t *testing.T) {
 	v := DetectVars()
 	assert.Equal(t, "", v.BuildID)
 }
+
+//nolint:paralleltest // mutates environment variables
```

Comment:
I don't think this comment is necessary anymore.  The linter should now autodetect when `t.Setenv` is used.

Reply:
Nice -- removed!